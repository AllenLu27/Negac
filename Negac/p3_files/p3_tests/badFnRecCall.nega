void fn(int a, void b) {
  fn(1, 2);
}
