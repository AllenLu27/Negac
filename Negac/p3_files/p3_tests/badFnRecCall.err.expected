FATAL [1,21]: Invalid type in declaration
FATAL [2,3]: Undeclared identifier
Semantic analysis failed
